File: API/iam/API/urls.py
from django.urls import path
from .endpoints import login
from .endpoints import logout
from .endpoints.profile import profile
from .endpoints import signup
from .endpoints.users import users
from .endpoints.user import user

urlpatterns = [
    path('login', login.login),
    path('logout', logout.logout),
    path('profile', profile.profile),
    path('signup', signup.signup),

    path('users', users.users),
    path('user', user.user)

    # games

]

File: API/home/src/lib/internalAPI/urls.js
"use strict";

module.exports = {
    iam: {
        hostname: 'api-iam',
        port: 8000,
        endpoints: {
            login: '/login',
            logout: '/logout',
            profile: '/profile',
            signup: '/signup'
        }
    },
    games: {
        hostname: 'api-games',
        port: 9000,
        endpoints: {
            games: '/games',
            game: '/game',
            load: '/load',
            save: '/save'
        }
    }
}
